src/templates/KanbanBoard/Layout.tsx

```diff
@@ -71,28 +71,27 @@ export function Layout() {
   };
 
   return (
-    <Box margin="4">
-      <Header
-        handleCreateRandomTasks={handleCreateRandomTasks}
-        onOpenStateDialog={onOpenStateDialog}
-      />
-      <Heading mx="auto" paddingBottom="2">
-        Board
-      </Heading>
-
-      <Flex direction={"row"} gap="2">
-        {sortedStates.map((value) => (
-          <KanbanColumn
-            key={value.name}
-            id={value.id}
-            title={value.name}
-            color={value.color}
-            onOpenCreateTaskModal={() => setStaskModalStateId(value.id)}
-            setEditTaskModalItem={setEditTaskModalTaskId}
-          />
-        ))}
-      </Flex>
-
+    <>
```

Comment:
Unnecessary Empty Tag. You can just make the Dialogs children of the <Box /> component